cli/src/main/scala/scala/scalanative/cli/ScalaNativeP.scala

```diff
@@ -32,24 +39,89 @@ object ScalaNativeP extends CaseApp[POptions] {
       System.err.println(s"Ignoring non existing path: $path")
     }
 
+    if (options.nirFiles) deserializeFiles(classpath, args.all)
+    else deserializeClassesOrObjects(classpath, args.all)
```

Comment:
I think more suitable names would be `deserializeFromPath` and `deserializeForName` (`deserialize` prefix might be committed as well since it's returning unit (alternatively it can return `Seq[Defn]` which we would then show here))